### DISCUSSION TITLE
Can't apply "border-spacing-3"

### DISCUSSION THREAD

**Discussion by jvalrog** [tailwindlabs/tailwindcss#8967] [Help] [answered]
I got this table in Vue3:
```
<template>
  <table class="bg-inherit border-separate border-spacing-3">
    <tbody>
      <tr v-for="item of dataSource" :key="item.key">
        <td class="align-top">{{ item.key }}</td>
        <td class="align-top bg-neutral-700 text-neutral-100 rounded">
          {{ item.value }}
        </td>
      </tr>
    </tbody>
  </table>
</template>
```

And the styles applied are: **bg-inherit** and **border-separate**. 
**border-spacing-3** is totally missing from the table style as can be seen here:

![image](https://user-images.githubusercontent.com/456311/181224733-8b6d7d15-6f2d-4b5a-91bd-32af926e7553.png)

Any idea why is this happening?

I'm using **TailwindCss 3.0.24** and importing the **utilities** module alone.

**Comment by PositiveUser** [comment, marked as answer]
Tailwind CSS 3.0.24 is being used.
But, border spacing utilities are a new feature in version 3.1

https://tailwindcss.com/blog/tailwindcss-v3-1#border-spacing-utilities

**Reply by jvalrog** [reply]
thank you! it was driving me crazy haha
